speechbrain/run_wav2vec2_librispeech.sh

```diff
@@ -3,26 +3,87 @@
 export PYTHONPATH="..":$PYTHONPATH
 
 SOURCE="speechbrain/asr-wav2vec2-librispeech"
```

Comment:
The only speechbrain model currently submitted to the leaderboard (for which we'll do the full eval suite)